yw_website/website/views.py

```diff
@@ -25,6 +25,7 @@ def home(request):
     documents = paginator.get_page(page)
     return render(request, 'pages/home_page.html', { 'document_list': documents })
 
+@login_required()
 def my_workflows(request):
     # TODO: Handle unathenticated user? Right now will just load an empty table.
```

Comment:
if the `login_required` decorator handles unauthenticated users, we should remove this comment.

Reply:
@aniehuser done.